templates/cube/study/index.md

```diff
@@ -19,4 +19,22 @@ For additional practise, enroll in Canonical’s Study Labs. The labs feature vi
 
 A single purchase grants 90 days of access to the full set of Study Labs.
 
-<a class="p-button--positive">Purchase Study Lab access</a>
\ No newline at end of file
+<a class="p-button--positive js-study-button"><i class="p-icon--spinner u-animation--spin is-light"></i></a>
```

Comment:
Could you add a `u-hide` class with an empty href and have the JS remove the class on load?

Reply:
Can you add the words "Loading" inside the icon so there is some content please